apps/server/src/modules/group/controller/dto/request/class-filter-params.ts

```diff
@@ -0,0 +1,15 @@
+import { ApiPropertyOptional } from '@nestjs/swagger';
+import { IsEnum, IsOptional } from 'class-validator';
+
+export enum SchoolYearQueryType {
```

Comment:
Two things in one file?

Reply:
analogous to class-sort-params.ts in the same directory